frontend/src/components/pages/UserManagement/index.tsx

```diff
@@ -150,77 +157,85 @@ const UserManagementPage = (): JSX.Element => {
       <Text mb="40px" textStyle="desktopHeader2">
         User management
       </Text>
-      <HStack>
-        <InputGroup flex={4}>
-          <InputLeftElement pointerEvents="none">
-            <SearchIcon color="gray.300" />
-          </InputLeftElement>
-          <Input
-            enterKeyHint="enter"
-            onChange={(e) => setSearch(e.target.value)}
-            placeholder="Search for a user"
-            variant="customFilled"
-          />
-        </InputGroup>
-        <Select
-          flex={1}
-          color="hubbard.100"
-          onChange={(e: React.ChangeEvent<HTMLSelectElement>) => {
-            handleSelectFilter(e);
-          }}
-        >
-          {accountTypefilterOptions.map((option, key) => {
-            return (
-              <option key={key} value={option.value}>
-                {option.label}
-              </option>
-            );
-          })}
-        </Select>
-      </HStack>
-
-      <Table colorScheme="blackAlpha" mt="30px">
-        <Thead background="squash.100">
-          <Tr>
-            <Th color="black.100">Point of contact</Th>
-            <Th color="black.100">Company</Th>
-            <Th color="black.100">Email</Th>
-            <Th color="black.100">Phone number</Th>
-            <Th color="black.100">Account type</Th>
-            <Th color="black.100">Approvals</Th>
-            <Th color="black.100" />
-          </Tr>
-        </Thead>
-        <Tbody>
-          {tableData.map((user, key) => (
-            <Tr key={key}>
-              <Td>{user.pointOfContact}</Td>
-              <Td>{user.company}</Td>
-              <Td>{user.email}</Td>
-              <Td>{user.phoneNumber}</Td>
-              <Td>{user.accountType}</Td>
-              <Td>
-                {user.accountType === Role.VOLUNTEER &&
-                user.approvalStatus !== Status.APPROVED ? (
-                  <Button variant="approve" onClick={() => handleApprove(user)}>
-                    Approve
-                  </Button>
-                ) : (
-                  ""
-                )}
-              </Td>
-              <Td>
-                <IconButton
-                  backgroundColor="transparent"
-                  aria-label="Delete user"
-                  icon={<DeleteIcon color="hubbard.100" />}
-                  onClick={() => handleDeleteUser(user)}
-                />
-              </Td>
-            </Tr>
-          ))}
-        </Tbody>
-      </Table>
+      {dataError ? (
+        <Text>Something went wrong with loading the data, please refresh the page and try again!</Text>
+      ) : (
```

Comment:
change this to {dataError && <Text>Something went wrong with loading the data, please refresh the page and try again!</Text>}